install/0000_80_machine-config-operator_03_rbac.yaml

```diff
@@ -16,6 +16,23 @@ roleRef:
   name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
 ---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: service-account-openshift-machine-config-controller
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+subjects:
+- kind: ServiceAccount
+  name: machine-config-controller
+  namespace: openshift-machine-config-operator
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Um, how does this relate to the subject of this PR? Isn’t there some lesser privilege that could be used?

`clusterrole.yaml` is already adding access to the ITMS/IDMS data.

Reply:
removed